pom.xml

```diff
@@ -80,7 +80,7 @@
         <maven-javadoc-plugin.version>3.4.0</maven-javadoc-plugin.version>
         <spring-boot.version>2.6.12</spring-boot.version>
         <bc.version>1.70</bc.version>
-        <jackson.version>2.13.3</jackson.version>
+        <jackson.version>2.13.4</jackson.version>
```

Comment:
Ok, this fix the issue, but I see that there is spring boot so I would prefer to use their _bom_, where the newer version already is, see https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/2.6.12

Reply:
We can look into this later.